src/components/Home/HomeDetail.tsx

```diff
@@ -211,6 +213,10 @@ function HomeDetail(props: Props) {
       },
     },
     chart: {
+      stakedMkr: {
+        data: getStakedMkrData(data, chartFilters.stakedMkr),
```

Comment:
I would move it to the state and update it thru useEffect hook in order to prevent slow rendering.